README.md

```diff
@@ -36,27 +36,51 @@ Alternative values can be passed to the `<Timeline>` component,
 e.g. `<Timeline animations={false}` to disable animations (far more efficient,
 as otherwise boundingRect of every entry is checked on every scroll event).
 
+### Global
+
 name                  | default value            | description
 ----------------------|--------------------------|-----------------------
-paddingTop            | 50
+totalWidth            | detected screen width | The total width of the view
 mediaWidthMed         | 900
 mediaWidthSmall       | 700
 activeColor           | #F45B69
 color                 | black
-twoSidedOverlap       | 80 | negative overlap between items if two-sided
 animations            | true
+animationScrollDelay  | 100 | Delays triggering the animation when scrolling up or down
```

Comment:
The wording leaves it unclear if this a time-based or offset/space-based delay.

From the code I see that this is an offset delay of `100px`, I think it should
1. be mentioned in the description
2. possibly added to the variable name.